pkg/operator/starter.go

```diff
@@ -142,12 +142,19 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		ctx.EventRecorder,
 	)
 
+	certRotationBase := time.Duration(0)
+	if certRotationConfig, err := kubeClient.CoreV1().ConfigMaps(operatorclient.OperatorNamespace).Get("unsupported-cert-rotation-config", metav1.GetOptions{}); err == nil {
```

Comment:
better just ignore NotFound errors, but retry on different errors.

Reply:
and fail if retries fail as well.

Reply:
addressed